scripts/gen_vimdoc.py

```diff
@@ -123,11 +123,13 @@
             'vim.lua',
             'shared.lua',
             'uri.lua',
+            'ui.lua',
         ],
         'files': ' '.join([
             os.path.join(base_dir, 'src/nvim/lua/vim.lua'),
             os.path.join(base_dir, 'runtime/lua/vim/shared.lua'),
             os.path.join(base_dir, 'runtime/lua/vim/uri.lua'),
+            os.path.join(base_dir, 'runtime/lua/vim/ui.lua'),
```

Comment:
Should we have a separate docs file or is it okay to amend this to lua.txt?

Reply:
lua.txt is fine for now